recipes/orekit/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "orekit" %}
+{% set version = "9.0" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://www.orekit.org/git/orekit-labs/python-wrapper-main.git
+  git_tag: orekit-v9
+  sha256: {{ sha256 }}
```

Comment:
Can we please use a tarball instead of cloning the repo?

ref: https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos